Sources/Plasma/PubUtilLib/plInputCore/plInputDevice.cpp

```diff
@@ -292,15 +292,15 @@ void plMouseDevice::AddNameToCursor(const ST::string& name)
     if (fInstance && !name.empty())
     {
         plDebugText     &txt = plDebugText::Instance();
-        txt.DrawString(fInstance->fWXPos + 12 ,fInstance->fWYPos - 7,name);
+        txt.DrawString(fInstance->fWXPos + 12 * fScale, fInstance->fWYPos - txt.GetFontHeight() / 2,name);
```

Comment:
This "fixes" the mouse over nameplate on over players by vertically centering it with the cursor - but noticing on Windows thats not how it works. Name is slightly vertically below the cursor. This should probably be re-adjusted.